site/content/imgpkg/docs/latest/auth.md

```diff
@@ -36,7 +36,8 @@ You can explicitly specify credentials via command flags or associated environme
 
 As of v0.7.0+, `imgpkg` can also use following environment variables:
 
-- `IMGPKG_REGISTRY_HOSTNAME` to specify registry hostname (e.g. gcr.io, docker.io)
+- `IMGPKG_REGISTRY_HOSTNAME` to specify registry hostname (e.g. gcr.io, docker.io, https://gcr.io, docker.io/v2/)
+  - As of v0.18.0+ `IMGPKG_REGISTRY_HOSTNAME` also supports providing glob wildcards. for e.g. `*.*.docker.io` will match `bar.foo.docker.io`
```

Comment:
worth adding that path matching also works

Reply:
probably also worth mentioning how the order within env vars is figured out (eg globs end up being last kind of thing).

Reply:
good call out. done